### PR TITLE
Added new documentation of entities

### PR DESCRIPTION
**Pull request by osinmv**
I added 11 documentation posts of game entities
image
I tested my changes. I think that everything should be fine

### CODE REVIEW

**Comment by patrickmoc** [comment]
In the future, the description and title of the pull request should be more descriptive of what area of the documentation site has been worked on (I/E entities, guides, cvars). I'm not sure what the "(11 out of #)" is referring to. The commits are structured somewhat inconsistently as well with the first one only adding one file while the others add a whole bunch. Sometimes it is better to split your changes up into smaller more concise ones that can be more accurately described.

**Comment by osinmv** [comment]
Thanks for a review

File: _posts/entities/2020-07-13-momentum_generic_bomb.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+>**Explosion Sound** (sound&lt;**sound**&gt;)
+
+Name of the sound effect to play when exploding.
+
+## Input 
+
+>input Detonate(**void**)
+
+Force detonation.Force detonation.
```
Comment (Gocnak) [inline _posts/entities/2020-07-13-momentum_generic_bomb.md:34]: Duplicate sentence?

File: _posts/entities/2020-07-16-func_ladder.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+title: func_ladder
+category: entity
+tags:
+- func
+- map
+- basics
+---
+----
+
+Ladder. Players will be able to freely along this brush, as if it was a ladder. If you are using a model prop for the visual representation of the ladder in the map, apply the toolsinvisibleladder material to the func_ladder brush.
```
Comment (Gocnak) [inline _posts/entities/2020-07-16-func_ladder.md:11]: Actually, this entity is a default Source entity (https://developer.valvesoftware.com/wiki/Func_ladder) and does not need a momentum documentation page. It was in the FGD for parity, so this page can be deleted.

File: _posts/entities/2020-07-16-trigger_momentum_limitmovement.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: trigger_momentum_limitmovement
+category: entity
+tags:
+- trigger
+- player
+- map
+- ahop
+- bhop
+- track
```
Comment (Gocnak) [inline _posts/entities/2020-07-16-trigger_momentum_limitmovement.md:10]: You can change the track tag out for `limit` and `restrict`

File: _posts/entities/2020-07-16-trigger_momentum_multihop.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: trigger_momentum_multihop
+category: entity
+tags:
+ - trigger
+ - map
+ - track
```
Comment (Gocnak) [inline _posts/entities/2020-07-16-trigger_momentum_multihop.md:7]: You can change the track tag to instead be `bhop`

**Comment by osinmv** [comment]
I fixed issues proposed by Gocnak.
I also double checked new posts with entitiy document. I wanted to be 100% sure that flags are definitely correct.

**Comment by osinmv** [comment]
I suppose that now it is ready.

File: _posts/entities/2020-07-16-trigger_momentum_multihop.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+>**Remote Destination** (target&lt;**target_destination**&gt;)  
+
+The entity specifying the point to which the player should be teleported.
+
+>**Stop player on teleport** (stop&lt;**choices**&gt;)  
+ - **0**: False
+ - **1**: True
+
+The defaule value is 1.
```
Comment (Gocnak) [inline _posts/entities/2020-07-16-trigger_momentum_multihop.md:31]:
```suggestion
The default value is 1.
```

File: _posts/entities/2020-07-16-trigger_momentum_onehop.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+>**Remote Destination** (target&lt;**target_destination**&gt;)  
+
+The entity specifying the point to which the player should be teleported.
+
+>**Stop player on teleport** (stop&lt;**choices**&gt;)  
+ - **0**: False
+ - **1**: True
+
+The defaule value is 1.
```
Comment (Gocnak) [inline _posts/entities/2020-07-16-trigger_momentum_onehop.md:33]:
```suggestion
The default value is 1.
```

File: _posts/entities/2020-07-15-trigger_momentum_teleport_progress.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+>**Remote Destination** (target&lt;**target_destination**&gt;)  
+
+The entity specifying the point to which the player should be teleported.
+
+>**Stop player on teleport** (stop&lt;**choices**&gt;)  
+ - **0**: False
+ - **1**: True
+
+The defaule value is 1.
```
Comment (Gocnak) [inline _posts/entities/2020-07-15-trigger_momentum_teleport_progress.md:32]:
```suggestion
The default value is 1.
```

Comment (osinmv) [reply _posts/entities/2020-07-15-trigger_momentum_teleport_progress.md]: I fixed the typos

**Pull request merged**